deepchem/models/torch_models/ferminet.py

```diff
@@ -190,7 +187,7 @@ def calculate_nuclear_nuclear(self,) -> torch.Tensor:
              self.nuclear_charge.unsqueeze(1)),
             posinf=0.0,
             neginf=0.0)
-        potential = torch.sum(potential) / 2
+        potential = (torch.sum(potential) / 2).unsqueeze(0)
```

Comment:
changing the format of of potential energy functions to calculate energies per batch, as hessians' and jacobians' energy too are per batch